py/server/tests/test_numba_guvectorize.py

```diff
@@ -66,7 +66,10 @@ def g(x, dummy, res):
             res[0] = min(x)
             res[1] = max(x)
 
-        t = empty_table(10).update(["X=i%3", "Y=i"]).group_by("X").update("Z=g(Y,dummy)")
+        # convert dummy to a Java array
+        # TODO this is a hack, we might want to add a helper function for QLP to call to get the type of a PyObject arg
+        j_array = dtypes.array(dtypes.int64, dummy)
+        t = empty_table(10).update(["X=i%3", "Y=i"]).group_by("X").update("Z=g(Y,j_array)")
```

Comment:
Ideally, this should work without the hack... but making it "just work" has some other problems.  Need to give this some thought.  In the multidim array world, this could do a zero copy buffer handover.  Without that bit, there could be various bad outcomes.

Reply:
https://github.com/deephaven/deephaven-core/issues/5278